Packs/ExabeamSecurityOperationsPlatform/Integrations/ExabeamSecOpsPlatform/ExabeamSecOpsPlatform.yml

```diff
@@ -577,30 +569,79 @@ script:
       - Replace
     - name: interval_in_seconds
       description: The interval in seconds between each poll.
-      defaultValue: 30
+      defaultValue: '30'
     - name: timeout
       description: The timeout in seconds until polling ends.
-      defaultValue: 600
+      defaultValue: '600'
     - name: tracker_id
       description: Specify the tracker ID from an upload request whose progress you want to track.
       hidden: true
     - name: hide_polling_output
       hidden: true
       description: Suppresses the output of polling operations to reduce clutter in logs.
     polling: true
+  - arguments:
```

Comment:
redefine args (that you don't send straight away to the api request) in camel case format.